### ISSUE TITLE
CLI Eval

### ISSUE THREAD

**Issue by acook** [munificent/wren#389]
This is a quite small thing, but most interpreters have it. 

~~~shell
$ wren -e 'System.print("Hello, world!")'
~~~

It seems simple enough (famous last words, right?) to add in basic support for it by breaking the `runFile` function into the part that actually loads the file and the part that runs the VM (probably in a new function called `runVM`), and then looking for the flag on the commandline (maybe just seeing if `argc == 3` and taking `argv[2]` for now and not worrying about a full commandline parser).

If this sounds like something you want I can probably do this myself and submit a pull request.

**Comment by minirop** [comment]
that sound nice, but I would go with solid foundations (like using getopt) and not quickly hacking a non-scalable thing.

**Comment by acook** [comment]
I'm ambivalent to the commandline option solution really.

Likely, getopt (or equivalent) could be added with a simple `--help` and `--version` flags, because right now wren can't do either of those. But that's beyond the scope of this issue imho.

**Comment by munificent** [comment]
Yeah, this sounds reasonable to me too. Figuring out how to actually parse the command line options is a tricky one. My hunch is that `getopt()` is a little too primitive. `getopt_long()` may not be portable enough.

Right now, the command line options are simple enough that we could probably just hand code them in C like we do for `--help` and `--version`.
